Source/WebKit/NetworkProcess/cocoa/NetworkDataTaskCocoa.mm

```diff
@@ -593,6 +594,19 @@ static float toNSURLSessionTaskPriority(WebCore::ResourceLoadPriority priority)
     if (m_failureScheduled)
         return;
 
+    CheckedPtr session = m_session.get();
+    if (session) {
+        CheckedPtr storageSession = session->networkStorageSession();
+        if (storageSession && storageSession->cookiesVersion() < m_requiredCookiesVersion) {
+            RELEASE_LOG(Loading, "NetworkDataTaskCocoa::resume is delayed because cookies version is too low");
+            storageSession->setCookiesVersionChangeCallback(m_requiredCookiesVersion, [weakThis = ThreadSafeWeakPtr { *this }] {
```

Comment:
Please add RELEASE_LOGGING so we can have a chance to debug things in sysdiagnoses if this misbehaves. In case of bug, this could easily cause unexplained load hangs.

Reply:
hmm this has `RELEASE_LOG(Loading,`, or did you suggest logging more information?

Reply:
Do we have logging when the load is actually unblocked by the cookie version upgrading?

Reply:
oh you mean in the callback, ok will add